src/so_vits_svc_fork/gui.py

```diff
@@ -255,10 +268,19 @@ def main():
             [
                 sg.Text("Input audio path"),
                 sg.Push(),
-                sg.InputText(key="input_path"),
+                sg.InputText(key="input_path", enable_events=True),
                 sg.FileBrowse(initial_folder=".", key="input_path_browse"),
                 sg.Button("Play", key="play_input"),
             ],
+            [
+                sg.Text("Output audio path"),
+                sg.Push(),
+                sg.InputText(key="output_path"),
+                sg.FileSaveAs(
+                    initial_folder=".",
+                    key="output_path_browse",
+                    file_types=(("WAV", ".wav"), ("MP3", ".mp3"))),
```

Comment:
I think more file types are supported.

Reply:
The file save output will now show all supported audio encoding types:

<img width="884" alt="Screenshot 2023-04-22 163506" src="https://user-images.githubusercontent.com/5108856/233812120-8bb988ab-821d-473c-930e-9ae701bead71.png">

At inference time the output extension will also be checked to ensure it is supported:

<img width="267" alt="Screenshot 2023-04-22 164421" src="https://user-images.githubusercontent.com/5108856/233812135-acefb953-7bb0-406f-96a6-15d1429d9a3b.png">